driver/src/main/java/org/neo4j/driver/internal/util/ServerVersion.java

```diff
@@ -54,6 +54,12 @@ public static ServerVersion version( Driver driver )
         }
     }
 
+    public static ServerVersion version( Session session )
```

Comment:
I think it is better to inline this method in `ParametersIT` because it is only used there